src/plugins/intel_gpu/src/graph/col2im.cpp

```diff
@@ -45,6 +38,26 @@ std::vector<layout> col2im_inst::calc_output_layouts(col2im_node const& node, ke
     auto output_type = desc->output_data_types[0].value_or(input_layout.data_type);
     auto output_format = input_layout.format;
 
+    auto reshaped_input = input_layout;
+    if (input_layout.is_static() && input_layout.get_rank() >= 4) {
+        bool is_batched = true;
+        auto num_blocks_l = input_layout.spatial(1);
+        if (num_blocks_l == 1 && !validate_num_blocks(impl_param, input_layout.spatial(1))) {
+            is_batched = false;
+            num_blocks_l = input_layout.feature();
+        }
+
+        const auto batch = is_batched ? input_layout.batch() : 1;
+        const auto num_elements = is_batched ? input_layout.feature() : input_layout.batch();
+
+        if (is_batched)
+            reshaped_input.set_partial_shape({batch, num_elements, num_blocks_l});
+        else
+            reshaped_input.set_partial_shape({num_elements, num_blocks_l});
+    } else if (input_layout.is_dynamic()) {
+        OPENVINO_ASSERT("Col2Im only suppports static shape.");
```

Comment:
wrong usage of `OPENVINO_ASSERT`. It should be some like below
`OPENVINO_ASSERT(input_layout.is_static(), "col2im supports static shape only.");`

Reply:
Thanks. Fixed